src/screens/MessageScreen.tsx

```diff
@@ -116,24 +150,26 @@ export function MessageScreen({
               style={styles.listItemView}
               descriptionNumberOfLines={1}
               descriptionStyle={
-                user.hasUnread
+                node
                   ? [styles.listItemSubtitleText, styles.newMessageText]
                   : styles.listItemSubtitleText
               }
-              left={(props) =>
-                renderLeft(props, {
-                  _id: user.userId,
-                  id: user.userId,
-                  name: user.displayName,
-                  avatar: user.picture,
-                  isUnread: user.hasUnread,
-                })
+              left={() =>
+                renderLeft(
+                  others.filter((profile) => node.userIds.includes(profile.id)),
+                  node.hasUnread,
+                )
               }
-              title={user.displayName}
-              description={`${user.isCreatedBySelf ? styles.selfPrefix : ''} ${
-                user.lastMessage
-              }`}
-              right={() => renderRight(user.lastMessageTime)}
+              title={others
+                .filter((profile) => node.userIds.includes(profile.id))
+                .map((profile) => profile.profile.displayName)
+                .join(', ')}
+              description={`${
+                node.latestMessageUserId === userData?.id
+                  ? styles.selfPrefix
+                  : ''
+              }${node.latestMessageText}`}
```

Comment:
This should use i18n to combine the parts of the message. I also wonder if the prefix is necessary. Apple just displays the last message in the preview.

Reply:
FWIW it was in our designs and I've enjoyed it from the perspective of testing but I'm certainly not set on it. If you think we should drop it, just say the word.

Reply:
Used i18n to combine the parts of a message but happy to remove the prefix if we think it isn't helpful.

Reply:
Let's keep it then!